tests/test_ui_healer.py

```
@@ -186,3 +188,229 @@ def test_healer_alias_does_not_override_canonical_when_both_present():
+def _qr_items(n: int):
+    return [{"label": f"Category {i}"} for i in range(n)]
+
+
+def test_quickreplies_cap_is_twelve():
+    """Lock the chiclet cap: 12 items validate, 13 do not."""
+    assert QUICK_REPLIES_MAX_ITEMS == 12
+    validate_props("QuickReplies", {"items": _qr_items(12)})  # no raise
+    raised = False
+    try:
+        validate_props("QuickReplies", {"items": _qr_items(13)})
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)